.gitignore

```
@@ -1,3 +1,4 @@
 .idea
 .editorconfig
-main
\ No newline at end of file
+main
+vendor/*
```

This can just be `vendor/` to ignore the directory itself rather than the files inside it.

```suggestion
vendor
```